src/tests/DataLayer.test.js

```diff
@@ -718,60 +711,6 @@ test('listener off: unregister multiple handlers', () => {
 // Invalid: data, event, listeners
 // -----------------------------------------------------------------------------------------------------------------
 
-test('invalid data', () => {
-  const data = {
-    page: {
-      id: '/content/mysite/en/products/crossfit',
-      siteLanguage: 'en-us',
-      siteCountry: 'US',
-      pageType: 'product detail',
-      pageName: 'pdp - crossfit zoom',
-      pageCategory: 'womens > shoes > athletic'
-    }
-  };
-  dataLayer.push({
-    data: data,
-    invalid: 'invalid'
-  });
-  expect(dataLayer.getState()).toStrictEqual({});
-});
-
-test('invalid event', () => {
-  const data = {
-    page: {
-      id: '/content/mysite/en/products/crossfit',
-      siteLanguage: 'en-us',
-      siteCountry: 'US',
-      pageType: 'product detail',
-      pageName: 'pdp - crossfit zoom',
-      pageCategory: 'womens > shoes > athletic'
-    }
-  };
-  dataLayer.push({
-    event: 'clicked',
-    data: data,
-    invalid: 'invalid'
-  });
-  expect(dataLayer.getState()).toStrictEqual({});
-});
-
-test('invalid listener on', () => {
```

Comment:
I'd keep the code for invalid listeners until we update the logic with add/remove event listeners.

Reply:
I added the unit test back again but skip it for now.